packages/lesswrong/components/icons/SettingsButton.tsx

```diff
@@ -27,6 +27,11 @@ const styles = (theme: ThemeType): JssStyles => ({
   },
   blackLabel: {
     color: theme.palette.text.primary,
+  },
+  whiteLabel: {
+    [theme.breakpoints.up('lg')]: {
+      color: theme.palette.text.alwaysWhite
+    }
```

Comment:
White is the default so this has unexpected consequences such as changing the colour of the "sorted by" button in the dialogue to add a draft post to a sequence (there are probably other places too, but this is the first one I found):

<img width="660" alt="Screenshot 2024-03-27 at 11 18 06" src="https://github.com/ForumMagnum/ForumMagnum/assets/5075628/d0f830a3-4f24-4e09-b88e-41e50ca0f212">

This also means all the places where you're explicitely adding `color="white"` or `color={isLW ? "white" : undefined}` aren't actually doing anything as they're already using the white color anyway.

Reply:
Oh, it's the default, oops. Will fix.

Reply:
(Fixed)

Reply:
This is still broken, but just in a different way:
<img width="649" alt="Screenshot 2024-03-27 at 23 16 12" src="https://github.com/ForumMagnum/ForumMagnum/assets/5075628/7761a899-0c3a-4c6f-a5d4-4a4ae1a00f67">

The correct way to fix this is probably to add a third "grey" variant that doesn't apply either the "black" or "white" classes but just uses the current `grey[600]` value, and this should be the default (as it currently is).